lib/ansible/modules/find.py

```diff
@@ -154,6 +154,15 @@
             - When doing a C(contains) search, determine the encoding of the files to be searched.
         type: str
         version_added: "2.17"
+    max_matches:
+        description:
+            - Set the maximum number of matches to make before returning.
```

Comment:
more like:
The maximum number of matching paths returned, after finding this many the find action will stop looking.

Reply:
Updated